test/parallel/test-net-client-bind-twice.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// This tests that net.connect() from a used local port throws EADDRINUSE.
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+const server1 = net.createServer(common.mustNotCall());
+server1.listen(common.PORT, '127.0.0.1', common.mustCall(() => {
+  const server2 = net.createServer(common.mustNotCall());
+  server2.listen(common.PORT + 1, '127.0.0.1', common.mustCall(() => {
+    const client = net.connect({
+      host: server1.address().address,
```

Comment:
`common.localhostIPv4` here as well because of https://github.com/nodejs/node/issues/14900